docker-compose.prod.yml

```diff
@@ -19,6 +19,10 @@ version: '3.7'
 services:
   caddy:
     image: caddy:2
+    env_file:
```

Comment:
Ah, could you also update `docker-compose.desktop.yml` with these same fixes?

Reply:
That's the file downloaded for other OSes

Reply:
Yeah, I guess we also would need to change the public port too, so they both adhere to configuration.